src/filters/filter_fader.hpp

```diff
@@ -90,6 +103,10 @@ namespace dmxfish::filters {
                     this->storage.amber = col_ptr->get_amber_value();
                     this->storage.uv = col_ptr->get_uv_value();
                 }
+                if constexpr (MODE != bank_mode::DIRECT_INPUT_MODE) {
+                    if(this->storage.global_main_enabled)
+		        this->storage.color.iluminance *= (get_iomanager_instance()->get_global_illumination() / 65566);
+                }
```

Comment:
i did not understand yet. what does the main brightness (is this the global_illumination) effect?
should it effect the color and the dimmer, if a fixture contains both? Or do we handle only the color, and only in the last step we manipulate the dimmer if it contains, but the color illumination is set to 1 again?

Reply:
We have the main brightness fader that can be used to mix in the main brightness to reduce or increase the overall brightness at once. As there is no way to guess the correct behavior for this, we implement it with the filters. It effects usually the dimmer but this might not always be the case. As a consequence i made the honoring of this setting optional and let the UI handle the fixture and effect specific aspects.